src/fsharp/FSharp.Core.nuget/FSharp.Core.4.1.xxx.nuspec

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <!-- rename to FSharp.Core when done -->
+        <id>FSharp.Core</id>
+        <description>
+            FSharp.Core redistributables from Visual F# 4.1.0
+                Supported Platforms:
+                    .NET 2.0                    (net20)
+                    .NET 4.0                    (net40)
+                    .NET 4.5+                   (net45)
+                    .NET Portable Profile 7     (portable-net45+netcore45)
+                    .NET Portable Profile 47    (portable-net45+sl5+netcore45)
+                    .NET Portable Profile 78    (portable-net45+netcore45+wp8)
+                    .NET Portable Profile 259   (portable-net45+netcore45+wpa81+wp8)
+                    .NET Standard 1.6           (netstandard1.6)
+                    MonoAndroid, MonoTouch, Xamarin.iOS (portable-net45+monoandroid10+monotouch10+xamarinios10)
+                    XamarinMac mobile           (xamarinmac20)
+        </description>
+        <language>en-US</language>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <version>$version$</version>
+        <authors>$authors$</authors>
+        <licenseUrl>$licenseUrl$</licenseUrl>
+        <projectUrl>$projectUrl$</projectUrl>
+        <tags>$tags$</tags>
+        <dependencies>
+            <group>
+                <dependency id="System.ValueTuple" version="4.3.0" />
+            </group>
+            <group>
```

Comment:
This is probably not a great choice.  For example, System.ValueTuple is now at 4.3.1.  In July it will be at I believe 4.4.0.  FSharp.Core does not use System.ValueTuple.  And yet would probably need to reship with a dependency on the higher version.  I suppose twice more by July.

I think it makes sense to remove this dependency, and developers who use struct tuples on the desktop make an explicit reference.

In the target files for SDK builds we do a wildcard reference to 4.*.  In projects in VS we reference the explicit package.  So now I think we now do different things everywhere.  Oh good grief ... what a mess.
At least if we do the reference in the targets file it can be manually disabled if there is a need to.  I shall remove it from here.

Hmmm stream of consciousness on GitHub ... weird.